source/features/stop-pjax-loading-with-esc.tsx

```diff
@@ -0,0 +1,66 @@
+import onetime from 'onetime';
+import select from 'select-dom';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+
+const progressLoaderLoadingClass = 'is-loading';
+
+function fixProfileNavAndTimeline() {
+	for (const stickyElement of select.all('.js-sticky:not(.is-stuck)')) {
+		stickyElement.removeAttribute('style');
+	}
+}
+
+function init() {
+	const progressLoader = select('.progress-pjax-loader')!;
+
+	window.addEventListener('keydown', event => {
+		if (event.key === 'Escape' && progressLoader.classList.contains(progressLoaderLoadingClass)) {
```

Comment:
Can you move all listeners out of `init`?

Also let's use an early return (`if (!) {return}`) to limit the indentation further